mftf/2.3/test/actions.md

```diff
@@ -1535,9 +1535,12 @@ Attribute|Type|Use|Description
 <pressKey userInput="a" selector="#targetElement" stepKey="pressA"/>
 ```
 
+_**parameterArray**_ attribute has to begin with `[` and end with `]`, array() syntax is not supported.
```

Comment:
The _**parameterArray**_ attribute must begin with `[` and end with `]`. An array() syntax is not supported.

Highlight array() . (Not sure if you use backticks or italics.)

Reply:
:ok: